stories/data/geoChart.ts

```diff
@@ -1156,3 +1158,56 @@ export function getExecutionResponse(
         dimensions: resultDimensions,
     };
 }
+export function getGeoConfig(
+    isWithLocation = false,
+    isWithSegmentBy = false,
+    isWithTooltipText = false,
+    isWithSize = false,
+    isWithColor = false,
```

Comment:
please refactor these parameters into json object. 
This makes code easier to maintain

Reply:
fixed